packages/web/src/components/list/MultiDropdownList.js

```diff
@@ -153,12 +158,12 @@ class MultiDropdownList extends Component {
 			}
 		});
 		// Treat defaultQuery and customQuery as reactive props
-		if (!isIdentical(this.props.defaultQuery, prevProps.defaultQuery)) {
+		if (!isQueryIdentical([], this.props, prevProps, 'defaultQuery')) {
```

Comment:
@ShahAnuj2610 In the below code, you're passing value but here you're passing an empty array as value.

Reply:
@bietkul an open question. We're updating query with `[]` value, so should that value be passed or currentValue?

https://github.com/appbaseio/reactivesearch/blob/87a201d95d396944dc85e6d8686a9593ddef1d5f/packages/web/src/components/list/MultiDropdownList.js#L163

Reply:
Use `currentValue`, it'll be updated when you set the `query`.

Reply:
@bietkul fixing it in all components. Since all of them are either passing `''` or `[]` in **updateQuery**